bot.js

```diff
@@ -43,6 +41,29 @@ client.on('message', msg => {
             msg.reply(res);
         }).catch(() => { msg.reply("Movie not found!"); });
     }
+
+    if (cmd.startsWith(cli.WEBHOOK)) {
```

Comment:
What is the pros using ```startsWith``` over ```===```

Reply:
Because it receives the arguments passed by the message (i.e., webhook name and avatar image URL) but has to start with `/createHook`

Reply:
startsWith is misleading cuz ```cmd``` is already processed and has the same length as any ```cli``` you want to compare.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Reply:
What would you suggest handling this?